modules/utxo-lib/src/networks.ts

```diff
@@ -19,7 +19,13 @@ forkId         src/script/interpreter.h  FORKID_*
  * @deprecated
  */
 const coins = {
+  /*
+   * The original Bitcoin Cash was renamed to bitcoin-abc, and bitcoin-cash-node forked from it.
+   * Later, bitcoin-abc is rebranded to ecash. Here, 'bch' corresponds to bitcoin-cash-node, and
+   * 'bcha' corresponds to ecash. Ref: https://github.com/bitcoin-cash-node/bitcoin-cash-node
+   * */
```

Comment:
Nice (and much needed) explanation. As I mentioned I didn't even know about the "ecash" rebranding.

I actually remember at the very beginning bitcoin cash was going by "BCC" but then they switched to "BCH" because BitConnect was already using "BCC" on binance or something.